multiqc/modules/picard/HsMetrics.py

```diff
@@ -108,7 +148,87 @@ def parse_reports(self):
             if s_name not in self.general_stats_data:
                 self.general_stats_data[s_name] = dict()
             self.general_stats_data[s_name].update( data[s_name] )
-    
-    # Return the number of detected samples to the parent module
+        data_table = _clean_table(data)
+        table_html = plots.table.plot(data_table, _get_headers(data_table))
+        if not isinstance(self.sections, list):
+            self.sections = list()
+        self.sections.append({
+                'name': 'HSMetrics',
+                'anchor': 'picard_hsmetrics',
+                'content': table_html})
+        self.sections.append(_add_target_bases(data))
+        self.sections.append(_add_hs_penalty(data))
+        # Return the number of detected samples to the parent module
     return len(self.picard_HsMetrics_data)
-    
\ No newline at end of file
+
+def _clean_table(data):
+    data_table = defaultdict(dict)
+    for s in data:
+        for h in data[s]:
+            if h.startswith("PCT"):
+                continue
+            if h.startswith("HS"):
+                continue
+            if h == "GENOME_SIZE":
+                continue
+            data_table[s][h] = data[s][h]
+    return data_table
+
+def _get_headers(data):
+    header = {}
+    for s in data:
+        for h in data[s]:
+            try:
+                float(data[s][h])
+            except:
+                continue
+            if h not in header:
+                this = {
+                'title': h.replace("_", " ").lower().capitalize(),
+                'description' : DESC[h] if h in DESC else "",
```

Comment:
I think `None` instead of `""`. If it's `None` I think it uses the title instead or something sensible? That's off the top of my head though, may make no difference.